spec/mailers/teacher_mailer_spec.rb

```diff
@@ -36,4 +47,17 @@
     expect(email.to[0]).to eq("lmock@berkeley.edu")
     expect(email.body.encoded).to include("Short Long")
   end
+
+  it "Sends Request Info Email" do
+    teacher = teachers(:long)
+    email = TeacherMailer.request_info_email(teacher, "Request Reason")
+    email.deliver_now
+    expect(email.from[0]).to eq("contact@bjc.berkeley.edu")
+    expect(email.to[0]).to eq("short@long.com")
+    expect(email.subject).to eq("Request Info Email")
+    # Test appearance of first_name
```

Comment:
This is a fair comment, but it'd be better to do something like 
expect body to **not** include `"first_name"`, or to be safe: expect that a template includes that string, then assert it is missing from the rendered email.